packages/office-ui-fabric-react/src/components/SelectedItemsList/BaseSelectedItemsList.tsx

```diff
@@ -6,27 +6,44 @@ import { initializeComponentRef } from '../../Utilities';
 
 export interface IBaseSelectedItemsListState<T = any> {
   items: T[];
+  selection: Selection;
```

Comment:
Similar here, I don't see why this needs to be in state since it never changes.

Reply:
I think it needs to be in state because  it was potentially changed in componentWillReceiveProps. I agree that it doesn't seem like it needs to be there but I wanted to try to preserve it as is to help reduce friction for this checkin.